docs/sources/datasources/loki.md

```diff
@@ -87,6 +87,9 @@ Querying and displaying log data from Loki is available via [Explore]({{< relref
 
 A log query consists of two parts: log stream selector, and a log pipeline. For performance reasons begin by choosing a log stream by selecting a log label.
 
+#### Logs volume
+
+You can display logs volume histogram after running a log query. By default, the histogram is loaded only after clicking the "Load logs volume" option. Select the "Auto-load logs volume" option in the log options row to load it automatically after each query. This feature is available only in Explore.
```

Comment:
- [ ] Move docs to main Explore page (mention it's supported for Loki only at the moment)

Would it make sense to cross-reference to main Explore as well? I guess not everyone reading Loki docs will read Explore main docs.

Reply:
something I had in mind a while back was to start writing in each datasource a table outlining supported an unsupported features in explore, with a link to their docs, like:

| Feature | Supported | Notes |
|---|---|---|
| Logs search term highlight | | |
| Logs histogram | partial | histogram is based on returned logs and not on the full range |
| Show Context | yes | |


but never proposed it

Reply:
Sounds like a good idea. Maybe I can leave it in Loki docs for now and create a task to create such table?

Reply:
Cannot we embed some snippets? I feel like I would like the same text on multiple places in docs instead of having a link but also don't want to write it multiple times. Not sure if that would help here but I thought that would be useful.

Reply:
Since we are hiding it behind a feature toggle I'll remove the docs.